index.html

```diff
@@ -1,12 +1,233 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+    <title>KMarketer</title>
 </head>
+
 <body>
-    
+    <section class="header">
+
+        <div class="header__navbar">
+            <div class="header__logo">
+                <h3 class="header__h3">KMarketer</h3>
```

Comment:
Naming something after its tag is not recommended because once you change the tag you the previous naming won't be referring to the new naming and you will have to change  a lot of lines and even in JS files if you have used. Thus, refine the naming "header__h3". I would recommend something like "header__company-name"